### PR TITLE
Allow passing arguments to wait(fn), just like evaluate()

### PR DESCRIPTION
**Pull request by yoz**
This fulfills #356.

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -141,8 +141,8 @@ Wait for `ms` milliseconds e.g. `.wait(5000)`
 #### .wait(selector)
 Wait until the element `selector` is present e.g. `.wait('#pay-button')`
 
-#### .wait(fn)
-Wait until the `fn` evaluated on the page returns `true`.
+#### .wait(fn, arg1, arg2,...)
```
Comment (matthewmueller) [inline Readme.md]: should document that these are optional: `.wait(fn [, arg1, arg2,...])`

**Comment by matthewmueller** [comment]
looks good! just a minor documentation tweak and we should be good to go!

**Comment by yoz** [comment]
Done - also did this for the arguments to `evaluate` in the Readme.md.

**Comment by matthewmueller** [comment]
ah thanks good catch!

**Pull request merged**